### DISCUSSION TITLE
tagging entites, (has registry.assign(___) changed?)

### DISCUSSION THREAD

**Discussion by rudiHammad** [skypjack/entt#1039] [Q&A] [answered]
hi, 
I was investigating how to add a tag in entt and I came across this post from 2020

https://github.com/skypjack/entt/issues/419

But I had a peek to the definition and assign has now as a first and second arguments iterators
`void assign(It first, It last, const entity_type destroyed)`, so I am assuming that this has changed in the later version? I tried to track down where previous versions of Entt but I wasn't sure were to look.

The way I am assigning tag is with emplace. Is this correct?
` registry.emplace<entt::tag<"PlayerTag"_hs>>(playerEntity); `

thx

**Comment by skypjack** [comment, marked as answer]
`emplace` works for a single entity, while `insert` is meant for multiple entities.
`assign` was used to fill an existing registry with a range of entities but it's not required anymore since when the entity storage is a thing.
